vscode/npm-package/tsconfig.cjs.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
why do we need a cjs configuration?

Reply:
VsCode needs to use CJS. When we build, we build for both

Reply:
I though vscode had added support for esm, as long as our project/extension is the right type. Can we double check that? I'd rather not maintain twice the distribution factor.

Reply:
I've checked. VsCode, from this still open issue https://github.com/microsoft/vscode/issues/130367 , doesn't yet support esm. Once it does, the changes for us will be minimal because the cjs maintenance only happens at build time